cmd/crowdsec-cli/decisions.go

```diff
@@ -94,11 +96,21 @@ To list/add/delete decisions
 				log.Fatalf("failed to parse Local API URL %s : %v ", csConfig.LapiClient.Credentials.Url, err.Error())
 			}
 
+			scenarios := []string{}
+			for _, scenario := range cwhub.GetItemMap(cwhub.SCENARIOS) {
```

Comment:
same as before, use GetUpstreamInstalledScenariosAsString

Reply:
> same as before, use GetUpstreamInstalledScenariosAsString

yes, sure. But we have to get this merged before. This feature is not in this branch.